nirc_ehr/resources/data/cost_center.tsv

```diff
@@ -0,0 +1,132 @@
+value	title
```

Comment:
Is NIRC OK with these values being public?

Reply:
Yeah sounds like they are pretty arbitrary values as they explained it to us, however I was debating if it should ETL or TSV, so converted to ETL just to ensure no issues.